### PR TITLE
Update of main fr translation

### PR DESCRIPTION
**Pull request by bugeaud**
Correct some non french sounding sentenses.  One example welcomepage.title, before "fully featured" was translated as "entièrement en vedette"  (which is gibberish) and now "riche en fonctionnalités".

### CODE REVIEW

**Comment by gpatel-fr** [comment]
Hello @bugeaud 
While your translation is excellent and as it is would hugely improve the situation of french translation, I have a small nitpick, precisely with the translation of the absurd 'entièrement en vedette' by 'riche en fonctionnalité'. 
The accepted, common translation is 'complet'.
I think it carry more the meaning of fullness of 'fully-featured', that is, the product has all the essential features, nothing is missing. 
Also, 'complet' is much shorter and as such is better for a slogan. Système de vidéoconférence sécurisé, complet et gratuir' is sounding better in my opinion.

**Comment by bugeaud** [comment]
Hello @gpatel-fr,

Thanks for this feedback.

" Système de vidéoconférence sécurisé, complet et gratuit" (w/ typo fixed) was indeed my first translation attempt. But after a second thought, I felt that no one would actually write such a "commercial" sentense in French. 

#1 It sounds a tad too much bragging for French language
#2 Because "complet" in French means what it sounds : 100% (and not 99.999%). And if you make such a written statement, it has to be 100% in order not to be a deceiptive argument. That is the reason why in French, people would add more details on the applicable scope by adding a "pour" addendum next to it or with a star that link to the details. But such an approach was out of the translation context as it would need to bring more details in the text  which would make it diverge from the original version.

I think that "fully" here would be better translated as "comble". But I've found no way so far using that with "features" in a French way. Any ideas ?

An alternative I have in mind is : 
"Système de vidéoconférence sécurisé, accompli et gratuit."

Accompli is removing the ambibuity of the scope of the 100% complenetess by directly refering to the video conferencing world. It also removed the bragging and bring a bit more ethos to the sentense. Plus it is shorter, the way a slogan should be 😉

Rgs,
JB

**Comment by bugeaud** [comment]
> Hi, thanks for your contribution!
> If you haven't already done so, could you please make sure you sign our CLA (https://jitsi.org/icla for individuals and https://jitsi.org/ccla for corporations)? We would unfortunately be unable to merge your patch unless we have that piece :(.

Thanks for bringging this to my attention. It's done.  Rgs, JB

**Comment by gpatel-fr** [comment]
@bugeaud: about the deceptive statement, I think that you are waaaay to much optimistic about the rigor of french commercial law; you make it sound as if consumer rights were greater in France than in the USA !!!! since 'full-featured' as the same implied meaning as 'complet'.
in fact, french commercial law accept the right to exageration; first internet search brings this:

https://uncertainregardweb.wordpress.com/2017/08/28/le-droit-francais-de-la-consommation-admet-lexageration-commerciale-les-juristes-lappellent-le-dolus-bonus-ce-qui-est-plus-elegant-que-bobard/

In short, you have the right to claim that your magic cream will make my hair infinitely silky because nobody in his right mind will believe such a thing, but not that if will make my hair 42,5% more silky than without it because it's a definite statement that can right or true. 
The french lore has also a common latin motto: 'caveat emptor' (buyer beware). French law accepts much worse than hyperbole.
Your concerns about ethos are laufable but really I could name several french businesses that are touting themselves as 'open source' and it's let'say a *slight* exageration, IMO it's much more deceptive than 'complet' for jitsi-meet.

Anyway, If someone would be willing to be laughed out of court in France, the 'not really secure' or 'not really free (gratuit)' argument could be used just as well as 'not really full-featured' (complet)
So no I don't buy your argument.

**Comment by bugeaud** [comment]
Yes dolus bonus, is the magic wand of hawkers at your local fun fair. So it is not illegal, and you are free to don't "buy" things as you did with my #2 argument 🤭 But still #1 applies, in this version, to me, the sentense sound more like it could have beeing said by a hawker.

To sum it up, any of 3 new versions are acceptable to me.

We shall agree to disagree on which one is the best one for sure 😉

**Comment by gpatel-fr** [comment]
@bugeaud 

'We shall agree to disagree on which one is the best one for sure'

yes I already agreed fully that your version is far better than the existing one and as you did the work it's fair to consider that your choice is the best. Thanks for explaining your reasoning to me.

**Comment by damencho** [comment]
Could you please fix the conflicts, thanks.

**Comment by gpatel-fr** [comment]
@bugeaud 
Hello again
while looking one more time at your translation, I have realized that something has escaped me.
you have replaced in the liveStreaming section at the keyword 'failedToStart' the word streaming by 'diffusion' but you have kept the article 'Le' it should be 'La'. That's a very very small nitpick but I think we should agree on this one.

**Comment by luixxiul** [comment]
This needs rebase.

File: lang/main-fr.json [hunk 1]
```diff
@@ -363,19 +363,19 @@
...
+        "enterStreamKey": "Entrez votre clé de diffusion Youtube Live ici",
+        "error": "La diffusion a échouée. Veuillez réessayer ultérieurement.",
         "errorAPI": "Une erreur s'est produite lors de l'accès à vos diffusions YouTube. Veuillez réessayer de vous connecter.",
-        "errorLiveStreamNotEnabled": "La diffusion en direct n'est pas activée pour  {{email}}. Merci de l'activer ou de vous connecter avec un compte où elle est déjà activée.",
+        "errorLiveStreamNotEnabled": "La diffusion en direct n'est pas activée pour {{email}}. Merci de l'activer ou de vous connecter avec un compte où elle est déjà activée.",
         "expandedOff": "La diffusion en direct a été arrêtée",
         "expandedOn": "La conférence est en cours de diffusion sur YouTube.",
         "expandedPending": "La diffusion en direct a commencé...",
-        "failedToStart": "Le Streaming n'as pas réussi à démarrer",
+        "failedToStart": "Le diffusion n'as pas réussi à démarrer",
```
Comment (RichardFevrier) [inline lang/main-fr.json:378]: @bugeaud can you just change "Le diffusion" by "La diffusion" and ping someone for the merge request please ? :) Your work is excellent !

**Comment by bugeaud** [comment]
Conflict review done + Typo fixed (tnx Richard !).

File: lang/main-fr.json [hunk 2]
```diff
@@ -392,12 +394,12 @@
...
         "signIn": "Se connecter avec Google",
-        "signInCTA": "Connectez vous ou entrez votre clé de flux live provenant de Youtube.",
+        "signInCTA": "Connectez vous ou saisissez votre clé de diffusion Youtube.",
         "signOut": "Se déconnecter",
         "start": "Démarrer la diffusion en direct",
-        "streamIdHelp": "Qu'est-ce que c'est ?",
-        "unavailableTitle": "Le Streaming est indisponible",
-        "youtubeTerms": "Conditions d’utilisation des services YouTube",
+        "streamIdHelp": "Qu'est-ce que c'est ?",
+        "unavailableTitle": "La diffusion est indisponible"
```
Comment (RichardFevrier) [inline lang/main-fr.json:401]: @bugeaud you removed a comma at the end of the line ;)

**Comment by bugeaud** [comment]
Thanks to @gpatel-fr and @RichardFevrier for pointing the typos. There was indeed another missing comma in my merge as brought by the lint checks (doh). Checks have passed now.

**Comment by RichardFevrier** [comment]
Excellent! Anyone for the merge? @saghul :)

**Pull request merged**